.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" "qemu-system-aarch64" )
+stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/overlay2" )
```

Comment:
It would be safer to check the output of the following before adding the overlay entry:

```
$ storage_driver=$(docker info|grep "^Storage Driver"|cut -d: -f2-|tr -d ' ')
$ echo "'$storage_driver'"
'overlay2'
```

Reply:
You're right. Just randomly checking on another machine, the `Storage Driver` for docker would be the old `aufs`.